src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
```

Comment:
You've already started nicely to export typing to separate files, so it would be cool to continue here as well, and export the type for the `TodoInfo` component to the `Todo.ts` file and give this type a more convenient name (For example: `TodoInfoType`).